README.md

```diff
@@ -326,15 +336,6 @@ public void Configure(IApplicationBuilder app)
 
             builder.AddVR() // vr 'none'
                 .None();
-
-            // You can also add arbitrary extra directives: plugin-types application/x-shockwave-flash"
-            builder.AddCustomDirective("plugin-types", "application/x-shockwave-flash");
-
-            // If a new feature policy is added that follows the standard conventions, you can use this overload
-            // iframe 'self' http://testUrl.com
-            builder.AddCustomDirective("iframe") // 
-                .Self()
-                .For("http://testUrl.com");
```

Comment:
This is still applicable I think (in case not everything is supported)?

Reply:
Thanks, I removed this because it's not supported in the AddPermissionsPolicy method. Don't know if the spec supports this

Reply:
Added it back, this is AddCustomFeature in permissions

Reply:
Oops, seems like it's not really named properly, but cheers!